pkg/network/setup/configstate.go

```diff
@@ -71,24 +71,35 @@ func (c *ConfigState) Run(nics []podNIC, preRunFunc func([]podNIC) ([]podNIC, er
 			)
 		}
 	}
+	origNICCount := len(nics)
+	pendingNICCount := len(pendingNICs)
 	nics = pendingNICs
 
-	if len(pendingNICs) == 0 {
+	if pendingNICCount == 0 {
 		return nil
 	}
+	if origNICCount > pendingNICCount {
+		// This scenario should not occur, however, if in some edge case it does, it is only recorder for
```

Comment:
Won't we see this warning again in case we have `Absent` interfaces here?

Reply:
I think you are correct, this scenario is no longer relevant now.
We may reach here with network interfaces marked for removal and we ignore them later on.

Removing this part.
Done.